referrer-policy/no-referrer-policy/no-referrer-policy.subresource.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>This page reports back it's referrer URL to the parent frame</title>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body style="background:#f4f4f4;">
+    <h2>This page reports back it's referrer URL to the parent frame.</h2>
+
+    <script>
+      // Display this page's location (iframe page - subresource to parent).
+      appendPageInfo("Child frame");
+
+      // Note: Read the ID from the query section of this page's URL.
+      // The ID is used to identify the sender of the message in the parent.
+      var pageInfo = {
+          id: document.location.search.replace("?id=", ""),
```

Comment:
This is a bit fragile, as it will break if `id` is not the only parameter. It's probably fine for now, but perhaps adding a `TODO` would be appropriate.